databaseconnect.py

```diff
@@ -146,7 +146,50 @@ def learn_question_response(H):
     cur.execute("SELECT id FROM statement_table ORDER BY id DESC")
     res = cur.fetchone()
     last_id = res[0]
-    cur.execute('UPDATE statement_table SET sentence=%s WHERE id=%s',(H,last_id))
+    cur.execute(f"UPDATE statement_table SET sentence='{H}' WHERE id={last_id}")
     db.commit()
     B = "Thank you! I have learnt this."
     return B,0
+
+def clear_table(table_name):
+    db = connection_to_database()
+    cur = db.cursor()
+
+    if table_name in ("question_table","statement_table"):
+        tables_to_be_cleaned = ("question_table","statement_table")
+        print("The following tables will be cleaned:\n")
+        for table in tables_to_be_cleaned:
+            describe_table(cur, table)
+
+        if input("Enter 'Y' to confirm cleaning of BOTH tables: ") in ("Y","y"):
+            for table in tables_to_be_cleaned:
+                cur.execute("DELETE FROM {table}")
+            db.commit()
+            print("Tables cleaned successfully")
+        else:
+            print("Table cleaning skipped.")
+
+    else:
+        print("The following table will be cleaned:\n")
+        describe_table(cur, table_name)
+
+        if input("Enter 'Y' to confirm: ") in ("Y","y"):
+            print("Table cleaned successfully")
+            cur.execute(f"DELETE FROM {table_name}")
+            db.commit()
+        else:
+            print("Table cleaning skipped.")
+
+def describe_table(cursor, table_name):
```

Comment:
A function to replace this entire process makes more sense. Again, you can replace the multiple print statements here with a single f-string but that's just really nitpicking from me. Looks good 😄

Reply:
A good suggestion. 🙃 Nitpicking is indeed important for using best practices. 😅